docs/test_instructions_per_dependency_update.md

```diff
@@ -0,0 +1,859 @@
+  button (note that you need to use a non `A8C` account to be able to complete this flow).
+- Verify that the Google sign-in works, that you have successfully logged-in and are able to use the
+  app as expected.
+
+</details>
+
+-----
+
+### PlayServicesCodeScanner [[googlePlayServicesCodeScannerVersion](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/build.gradle#L1)] <a name="playservicescodescanner"></a>
+
+<details>
+  <summary>1. [JP] Scan Login Code</summary>
+
+Step.1:
+- Build and install the `Jetpack` app (note that you don't need a release build, a debug build will
+  suffice).
+- Login to the `Jetpack` app with a `WP.com` account (note that you need to use a non `A8C` account
+  and a non `2FA` enabled account).
+- Navigate to the `Me` screen (click on avatar at top-right).
+- (STOP)
+
+Step.2:
+- Head over to your desktop and open a web browser (note that using an incognito tab works best).
+- Browse to `wordpress.com` (note that if you are logged-in, log-out first).
+- Tap the `Log In` link (top-right).
+- Tap the `Login via the mobile app` link in the list of options below the main `Continue` button
+  (bottom-middle).
+- Verify you are on the `Login via the mobile app` view and `Use QR Code to login` is shown, along
+  with a QR code for you to scan.
+- (STOP)
+
+Step.3:
+- Head back to your mobile.
+- Tap the `Scan Login Code` item on the `Me` screen you are currently at.
+- Scan the QR code on the web browser.
+- Follow the remaining prompts on your mobile to login to WordPress on your web browser (desktop),
+  verify that you have successfully logged-in and are able to use WordPress as expected.
+
+</details>
+
+-----
+
+### Okio [[squareupOkioVersion](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/build.gradle#L1)] <a name="okio"></a>
+
+<details>
+    <summary>1. [JP/WP] Me Screen [GravatarApi.java + StreamingRequest.java]</summary>
+
+- Go to `Me` tab.
+- From the `Me` screen you are in, click on your profile's icon (`CHANGE PHOTO`).
+- Choose an image and wait for the `Edit Photo` screen to appear.
+- Crop the image and click the `done` menu option (top right).
+- Verify the image is updated accordingly.
+
+</details>
+
+-----
+
+### Zendesk [[zendeskVersion](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/build.gradle#L1)] <a name="zendesk"></a>
+
+<details>
+    <summary>1. [JP] Zendesk Update [Tickets & Contact Support]</summary>
+
+ℹ️ This test only works when testing with a normal, non-a8c user account.
+
+- Go to `Me` tab -> `Help` -> `Tickets`.
+- Verify that the `My tickets` Zendesk related screen opens and that it lists all of your tickets
+  (or non if you don't have any yet).
+- Go back to the `Help` settings screen and then click on `Contact Support`.
+- Verify that the `Help` Zendesk related screen opens and that you are prompt to `Send a message...`.
+- Type `Testing Zendesk X.Y.Z update, please ignore and thank you!` and then send your message.
```

Comment:
This step is outdated now. The `Contact support` button opens the chatbot. We need to tap the "Contact support" button after sending the first message to the bot.

Reply:
Awesome @irfano , thanks for that correction, I will add an extra step to tap on the `Contact support 📢` button, tested it and it  indeed works as you described, not quite outdated, just enhanced let me say. 😊

Reply:
Done: b1dd15b749c0b8fbf791814611b040dff373c5ca